pom.xml

```diff
@@ -464,6 +464,18 @@ THE SOFTWARE.
   </dependencies>
 
   <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <configuration>
+            <!-- Prevent the annoying ForkedBooter process from stealing window focus on Mac OS -->
+            <!-- Must be defined as argLine and not as system property to be used -->
+            <argLine>-Djava.awt.headless=true</argLine>
```

Comment:
Tis change will be useless when we'll use a new version of the parent including https://github.com/jenkinsci/plugin-pom/pull/22/commits/ed503f32bb1a1918865c4560363913c2253c282d